core/spec/models/spree/tax/tax_helpers_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+class DummyClass
+  include Spree::Tax::TaxHelpers
+
+  def valid_rates(item)
+    rates_for_item(item)
+  end
+end
```

Comment:
Instead of defining the class outside of the tests, can you do with `before` and `stub_const`?
```ruby
before do
  stub_const('DummyClass', (Class.new do
    include Spree::Tax::TaxHelpers

    def valid_rates(item)
      rates_for_item(item)
    end
  end))
end
```
The reason is that a class defined with the `class` keyword will leak between tests, while one defined with `stub_const` will be unloaded as soon as the test has finished executing.

Reply:
Thanks for the feedback, I've pushed the requested changes.